src/index.js

```diff
@@ -27,7 +28,7 @@ loader.pitch = function (request) {
     import {Comlink} from 'comlinkjs';
     ${multi ? '' : 'var inst;'}
     export default function f() {
-      ${multi ? 'var inst =' : 'inst = inst ||'} Comlink.proxy(require('!worker-loader?${JSON.stringify(options)}!${path.resolve(__dirname, 'comlink-worker-loader.js')}!${request}')());
+      ${multi ? 'var inst =' : 'inst = inst ||'} Comlink.proxy(require('!worker-loader?${JSON.stringify(options)}!${slash(path.resolve(__dirname, 'comlink-worker-loader.js'))}!${slash(request)}')());
```

Comment:
`request` comes from webpack and is already normalized to forward slashes.

```suggestion
      ${multi ? 'var inst =' : 'inst = inst ||'} Comlink.proxy(require('!worker-loader?${JSON.stringify(options)}!${slash(path.resolve(__dirname, 'comlink-worker-loader.js'))}!${request}')());
```